elements/pfe-dropdown/pfe-dropdown-item.ts

```diff
@@ -63,6 +71,12 @@ export class PfeDropdownItem extends LitElement {
       this.setAttribute('tabindex', '0');
     }
   }
+
+  protected _clickHandler() {
+    // Forward all click events to the elements in the lightdom.
+    // This fixes <a> tags from not being selected with keyboard events.
+    (this.firstElementChild as HTMLElement)?.click();
```

Comment:
this will click the link twice, right? should we guard here to make sure the target is `this`?

Reply:
Good call, added guard.